src/Robo/Plugin/Traits/DatabaseDownloadTrait.php

```diff
@@ -169,4 +170,34 @@ protected function s3RegionForSite(string $siteName): string
         }
         return $region;
     }
+
+    /**
+     * Sanitizes a file name for the Windows file system.
+     *
+     * @param string $fileName
+     *   The file name to sanitize.
+     *
+     * @return string
+     *   The sanitized filename.
+     */
+    public function sanitizeFileNameForWindows(string $fileName): string
+    {
+        if (PHP_OS_FAMILY === 'Windows') {
```

Comment:
I had no idea there were constants with this information. Very cool.

Reply:
Me neither, develop-by-Google FTW